_posts/2019-07-29-vim-basics.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: vim Basics
+date: 2019-07-29 22:28:02
+categories:
+author: bronzdoc
+---
+
+# VIM basics
+
+I would not bore you by telling you the history of vim, and the differences between vi and vim and all of its unix philosophy....
+If you are interested in that, you can give it a look [here](https://en.wikipedia.org/wiki/Vim_(text_editor).
+
+So all you need to know is that vim is a text editor... in my opinion one of vim strengths comes with how flexible it can be. It can be as simple or complex as you wish. It can serve you to edit a simple config file on a server or you can make it a complete code editing environment if you want.
+
+In this blog post I'll show you the basics of vim and the smallest amount of knowledge to make vim useful for yourself and yes I'll teach you how to exit ;)
+
+I'll asume you already have vim installed, if you are on a \*nix system you proably already have it.
+
+Open terminal and type: `vim vim-basic`.
+
+You'll see something like the folowing:
+```
```

Comment:
The formatting is off here. The code block doesn't end in the right place. See screenshot. 

<img width="880" alt="Screen Shot 2019-08-05 at 3 29 16 PM" src="https://user-images.githubusercontent.com/28606034/62489856-d864b700-b795-11e9-91c7-b370f22ae65c.png">